src/MSBuildLocator/DiscoveryType.cs

```diff
@@ -20,6 +20,11 @@ public enum DiscoveryType
         /// <summary>
         ///     Discovery via Visual Studio Setup API.
         /// </summary>
-        VisualStudioSetup = 2
+        VisualStudioSetup = 2,
+
+        /// <summary>
+        /// Discovery via dotnet (hostfxr).
+        /// </summary>
+        DotNetSdk = 4
```

Comment:
Why is this a flags enum? In practice, can't it only ever be one of these values?

Reply:
Oh, I see. Because it can be passed as an argument to limit the discovery. #Closed